### DISCUSSION TITLE
Installing GitHub App in Organization

### DISCUSSION THREAD

**Discussion by gabrielso** [community/community#106353] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I'm starting to work on a GitHub app, but I can only install it in my personal account. I can't find a way to install it in an Organization I own (I want to test some behaviors when my App interacts with organizations too).

When I go to my App settings, I can only see my personal account on the "Install App" section. It's installed, and if I go to "configure" the installation I can only give access to repositories under my personal account... I wish I could give the App access to a repository in an Organization.

Any pointers?

**Comment by gabrielso** [comment, marked as answer]
In case anyone needs this in the future, the app needs to be made "public" for this to be possible. This doesn't list the app in the Marketplace so probably safe for this stage.

**Comment by Kimau** [comment]
So I found you just need to transfer ownership to the Org and you can keep the app private. 
FYI in Sep 2025
